Alliance.Common/_Module/ModuleData/module_sounds_autogen.py

```diff
@@ -0,0 +1,49 @@
+﻿import os
+import xml.etree.ElementTree as ET
+
+def generate_xml(directory):
```

Comment:
What is the purpose of this Python script? :o 
Is it going to be run on client side by players?
If yes, are we sure Python will be installed on their computers ?

Reply:
It's only for us to help generate the module_sounds.xml as it can be quite tedious to do it manually.